app/src/config/fun.json

```diff
@@ -51,22 +60,24 @@
     "dash",
     "tcsh"
   ],
-  "Xorg GPU drivers": [
+  "Xorg GPU Drivers": [
     "xf86-video-intel",
     "nvidia-utils",
     "xf86-video-amdgpu",
     "xf86-video-nouveau",
     "xf86-video-ati",
-    "nvidia-390xx-utils"
+    "xf86-video-vmware",
+    "xf86-video-openchrome"
   ],
-  "Runtime Environment": [
+  "Runtime Environments": [
     "perl",
     "bash",
     "python",
     "lua",
     "ruby",
     "nodejs",
     "java-environment-common",
-    "php"
+    "php",
+    "rust"
```

Comment:
rust is not really a runtime environment nor a scripting languages. One could add a new section like "systems language" and add things like rust, go, gcc etc.

Reply:
Totally fair! I added a section and filled in a few that came to mind and seemed interesting. I left out ADA and Objective C since the stats are really boring. If there are any I'm missing let me know. You can [preview the list I made here](https://pkgstats.archlinux.de/compare/packages#packages=clang,gcc,ghc,go,nim,ocaml,rust,rustup,zig).

Notably an outliers here is `rust` and `rustup` should be part of the same metric, they provide the same thing just in a different way. Unfortunately it shows up separately in the stats which throws the relative graphs way off (e.g. trying to compare Rust with Golang). Having both on the graph at least lets people know there is an anomaly.